OpenRA.Mods.Common/ActorExts.cs

```diff
@@ -144,5 +144,13 @@ public static CPos ClosestCell(this Actor self, IEnumerable<CPos> cells)
 		{
 			return cells.MinByOrDefault(c => (self.Location - c).LengthSquared);
 		}
+
+		public static bool PreventsAutoTarget(this Actor self, Actor targeter)
+		{
+			if (self == null)
```

Comment:
It can be null actually, but I wouldn't bother checking for it since if somebody tried it's probably a bug and an exception would be better anyway.